docs/source/tasks/plr.rst

```diff
@@ -0,0 +1,11 @@
+Pupillary light reflex (PLR)
+================
```

Comment:
```suggestion
===================
```

Reply:
In Sphinx (which is the format of these `RsT` files) headings, the underlines need to has the same number of characters as the heading above them.